config/machine_readable/how-to-vote.yml

```diff
@@ -65,6 +65,10 @@ faqs:
         <li>permanently</li>
       </ul>
       <p>Arrange to <a href="/how-to-vote/voting-by-proxy?src=schema">vote by proxy</a> if there are under 2 weeks until election day and you have not made arrangements.</p>
+      <p>There’s a different form to <a href="https://www.eoni.org.uk/Vote/Voting-by-post-or-proxy">apply to vote by post in Northern Ireland</a>.</p>
```

Comment:
Is it worth adding `rel="noopener noreferrer"` to this link?

Reply:
This is only going to be presented as an 'enhanced' search result straight on Google – so probably less worried about referrals and security, we'd leave Google deal with that 😬